content/_includes/FeedbackWindow.md

```diff
@@ -0,0 +1,16 @@
+---
+---
+
+{{< trueimage src="/images/SCALE/23.10/FeedbackWindow.png" alt="The Feedback Window" id="2: The Feedback Window" >}}
+
+Use this window to submit a star rating and comments or suggestions for the current page.
+
+{{< hint type=note >}}
+The feedback window is used for general user experience feedback. 
+Submit bug tickets through the **File Ticket** link that opens the **File Ticket** form, also accessed from the **System Settings > General** screen **File Ticket** button. See [Filing an Issue Ticket in SCALE]({{< relref "fileissuescale.md" >}}) for further instructions.
```

Comment:
I stumbled upon another article in the Getting Started/GettingSupport/FileIssueSCALE.md article that we need to look at and maybe cross link either with a relref or update tags and taglists.